bin/pycbc_make_skymap

```diff
@@ -145,10 +147,34 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, ifar, ifos, thresh_SNR,
         command.append("--gps-end-time"),command.append(str(gps_end_time))
         command.append("--trigger-time"),command.append(str(np.round(trig_time,5)))
         command.append("--window"),command.append(str(window))
+        
 
-        command.append("--frame-type")
-        command.append(frame_types[ifo])
-
+        if custom_frame_files is None:
+            command.append("--frame-type")
+            command.append(frame_types[ifo])
+        else:
+            print("check if the segment in the custom frame file is safe") 
```

Comment:
Please use `logging.info(...)` instead of `print(...)`.

Reply:
Thank you. I have revised it accordingly.